### PR TITLE
CheckMark_Bug

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Fixed the check mark alignment bug
I also found an edge case bug with an earlier change I made with navigation header vertical constraints

### CODE REVIEW

File: ResearchUI/ResearchUI/iOS/Views/RSDCheckmarkView.swift [hunk 1]
```diff
@@ -146,7 +150,7 @@ fileprivate let defaultSize: CGFloat = 122
         _shapeLayer.frame = self.layer.bounds
         self.layer.cornerRadius = _cornerRadius ?? 0
         
-        _backgroundLayer.frame = self.layer.frame.insetBy(dx: self.backgroundInset, dy: self.backgroundInset)
```
Comment (dephillipsmichael) [inline ResearchUI/ResearchUI/iOS/Views/RSDCheckmarkView.swift:149]: self.layer.frame actually references the frame within this view's superview.  So in the case of both triggers and medication logging, the x: and y: were way off.

Comment (syoung-smallwisdom) [reply ResearchUI/ResearchUI/iOS/Views/RSDCheckmarkView.swift:149]: Can you please change to
:
```
_backgroundLayer.frame = self.layer.bounds.insetBy(dx: self.backgroundInset, dy: self.backgroundInset)
```

Thanks, and thanks for fixing. :)

Comment (dephillipsmichael) [reply ResearchUI/ResearchUI/iOS/Views/RSDCheckmarkView.swift:149]: FTFY

**Review by syoung-smallwisdom — APPROVED** [review]
Thanks! Looks good

**Comment by dephillipsmichael** [comment]
I can't merge, can you?

**Pull request merged**